Refactor(ui): rename page title “ЗМІ про нас” to “Ми у ЗМІ” (#720)

alisa-korniienko opened the pull request:
## Description

Renamed the page title “ЗМІ про нас” to “Ми у ЗМІ” and the English version “Media About Us” to the approved “Our media work” across the project.
Updated the page heading text and navigation links (header and footer) with the new title.
Also updated related references in the database (lf-dev.navigations).

## Type of change

- [ ] New feature
- [ ] Bug fix
- [ ] Documentation update
- [x] Refactoring / Tech debt
- [ ] CI/CD improvement
- [ ] Other: <!-- specify -->

## How to test

1. Open the app and verify that all occurrences of “ЗМІ про нас” were replaced with “Ми у ЗМІ”.
2. Check the header, footer, and page content — all should display the updated titles (“Ми у ЗМІ” / “Our media work”).
3. Confirm in MongoDB Compass that the document under lf-dev.navigations contains the updated labels.

## Video / Screenshots

<img width="469" height="315" alt="db" src="https://github.com/user-attachments/assets/3dfd8fcc-8782-4773-8567-f918440362bc" />
<img width="1905" height="792" alt="Снимок экрана 2025-10-23 192006" src="https://github.com/user-attachments/assets/d1630ee6-7e82-4929-979b-33aaa2fb8a9b" />
<img width="1893" height="1062" alt="m2" src="https://github.com/user-attachments/assets/8fba861a-ca3d-46dc-b259-793cbf5e0fb7" />
<img width="1898" height="547" alt="m1" src="https://github.com/user-attachments/assets/668428c3-90cd-41ef-ac9f-64cedcfe5e66" />


## Checklist

- [x] Code compiles and runs correctly
- [x] Tests pass successfully
- [x] Linting checks are clean
- [x] No Sonar issues
- [x] Documentation updated (if applicable)
- [x] All comments and requested changes addressed
- [x] Branch is up to date with the target branch
- [x] Linked issue/task included
- [x] Task moved to the review column on the board

---

alisa-korniienko committed:
refactor(ui): rename page title 'ЗМІ про нас' to 'Ми у ЗМІ' (#720)

alisabond commented:
> There are two locales so you should update english name as well

Yes, of course. I just wanted to clarify with the team which English translation to choose - "In the media" or "We in media". Thank you!

alisabond commented:
> None, "Us in media" sounds the best)



> > > There are two locales so you should update english name as well
> > 
> > 
> > Yes, of course. I just wanted to clarify with the team which English translation to choose - "In the media" or "We in media". Thank you!
> 
> None, "Us in media" sounds the best)

Thanks, I'll change it now)

alisa-korniienko committed:
feat: update media page title to 'Ми у ЗМІ' and 'Media About Us' to 'Our media work'

# Conflicts:
#	app/[lang]/(pages)/media-about-us/page.tsx
#	i18n/messages/en.json

alisa-korniienko committed:
refactor(ui): set media page h1 to 'Our media work'

alisa-korniienko committed:
Merge branch 'develop' into feature/720/replace-media-about-us-title

Iarynovskyi commented on i18n/messages/en.json:
```diff
@@ -155,10 +155,6 @@
       "name": {
         "composition": "All compositions"
-      },
-      "notFound": {
-        "title": "OoPs! NoThinG HeRe",
-        "description": "Our archivist cat has searched every drawer. Try another query"
```
Why you deleted it ? 
I think you did it by accident, because this translation was used in another component and now because of it, your docker build on PR is failed

alisa-korniienko replied:
You're absolutely right — this happened during a rebase.
There was a merge conflict, and when WebStorm tried to resolve it automatically, it accidentally removed a few lines from the i18n file.
I’ve already fixed it and restored the missing translations. Please check again — everything should be fine now!

alisa-korniienko committed:
refactor(i18n): restore missing keys and keep mediaAboutUs rename

bandvov committed:
Merge branch 'develop' into feature/720/replace-media-about-us-title
